tests/cases/fourslash/findAllReferencesJsDocTypeLiteral.ts

```diff
@@ -14,9 +14,3 @@
 //// function f(o) { return o.nested.[|great|]; }
 
 verify.rangesReferenceEachOther();
-
```

Comment:
This was the test that broke under the new assertion. I'm just removing a comment here that does nothing (it only has two slashes).

Reply:
Yeah, I must have left it from earlier testing and checked it in by mistake. But it would probably be good to retain this as a parser test.

Reply:
By "this" I meant the *above* code. This is a regular comment that doesn't test anything.